docs/topics/http/decorators.txt

```diff
@@ -97,3 +97,21 @@ caching based on specific request headers.
     into account when building its cache key.
 
     See :ref:`using vary headers <using-vary-headers>`.
+
+.. module:: django.views.decorators.cache
+
+Caching
+=======
+
+The decorators in :mod:`django.views.decorators.cache` control server and
+client-side caching.
+
+.. function:: never_cache(view_func)
```

Comment:
This function is mentioned in `topics/cache.txt` and `ref/contrib/admin/index.txt` -- we could make those references links using :func: